packages/e2e-tests/specs/editor/various/__snapshots__/links.test.js.snap

```diff
@@ -68,6 +68,6 @@ exports[`Links should contain a label when it should open in a new tab 1`] = `
 
 exports[`Links should contain a label when it should open in a new tab 2`] = `
 "<!-- wp:paragraph -->
-<p>This is <a href=\\"http://wordpress.org\\">WordPress</a></p>
+<p>This is <a href=\\"http://wordpress.org\\" target=\\"_blank\\" rel=\\"noreferrer noopener\\">WordPress</a></p>
```

Comment:
I believe this snapshot has been invalid in `trunk` for a while. This is because the test sets open in new tab to it's active state which should result in the attributes being added to the HTML.